proc/stack.go

```diff
@@ -14,9 +14,16 @@ func (nra NoReturnAddr) Error() string {
 }
 
 type Stackframe struct {
-	Location
-	CFA int64
-	Ret uint64
+	// Address the function above this one on the call stack will return to
+	Return Location
+	// Address of the call instruction for the function above this one on the call stack
```

Comment:
Change to: "Address of the call instruction that invoked the function above on the call stack."